.drone.star

```diff
@@ -917,7 +917,7 @@ config = {
 						'bash ./apps/user_ldap/tests/acceptance/setConfig.sh',
 						'php occ ldap:show-config',
 						'php occ ldap:test-config "LDAPTestId"',
-						'php occ user:sync "OCA\User_LDAP\User_Proxy" -m remove',
+						'php occ user:sync "OCA\\\User_LDAP\\\User_Proxy" -m remove',
```

Comment:
`\\` didn't work here so tried with `\\\` and surprisingly it worked.

Reply:
I guess there are 2 levels of escaping in the drone starlark extension on the server and somewhere in the internal yml and processing commands for execution. Special magic!